internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -20,22 +20,34 @@ import (
 
 type kafkaStatus string
 
+func (ks kafkaStatus) String() string {
+	return string(ks)
+}
+
+type deploymentType string
```

Comment:
The name seems too generic: deployment of what? can we be more specific? for example, `ManagedKafkaDeploymentType`, or `DataPlaneKafkaDeploymentType`?

Reply:
We are in the kafka context, I' say kafka deployment.

Reply:
changed to `managedKafkaDeploymentType`

Reply:
I wouldn't say kafka deployment. It's certainly ManagedKafka or DataPlaneKafka but not Kafka :thinking: